src/20240723_AaveV3Ethereum_TokenLogicKarpatkeyServiceProviderPartnershipPhase2/TokenLogicKarpatkeyServiceProviderPartnershipPhase2.md

```diff
@@ -0,0 +1,127 @@
+
+**Liquidity**
+
+We will continue to lead the operations of the Aave Liquidity Committee (ALC), overseeing the liquidity growth across different protocols. This includes
+
+- Adjusting liquidity incentives to maintain sufficient market depth;
+- Defining and executing direct liquidity incentives;
+- Leading initiatives that create utility for GHO;
+- Optimizing the utilization of the DAO's strategic voting assets;
+- Supporting GSM Integrations with aggregators;
+- Ensuring GSM are adequately funded;
+- Supporting future D3M, Facilitator, and Protocol Owned Liquidity (POL) initiatives; and
+- Support efforts to integrate GHO to Centralized Exchanges, and fund managers.
+
+**Growth**
+
+As GHO continues to grow beyond Ethereum, our efforts will expand to support those new deployments to ensure adequate liquidity and peg stability is maintained. We will continue to work with teams that build with GHO and advocate for the adoption of GHO wherever practical.
+
+When GHO is mature enough to support deploying Protocol Owned Liquidity, we will actively contribute to the creation of sGHO and balancing the needs for sGHO and stkGHO within the Aave ecosystem.
+
+Our goal is to keep the consistent growth rate and surpass the GHO borrowed supply of 180M by the end of this new term.
+
+### Aave Stack
+
+**Safety Module Management and Optimisation**
+
+The Safety Module (SM) currently presents exposure risks due to its reliance on the DAO's native token and its deployment solely on the Ethereum blockchain.
+
+With multiple instances of the SM being deployed, we will continue to perform the following:
+
+- Transition from AAVE emissions;
+- Introduce new categories;
+- Amend emission schedules; and,
+- Progressively align protocol exposure with SM holdings;
+
+**Aave Liquidity Network**
+
+With upcoming changes in the Aave product stack, including the Liquidity Network and the stablecoin's multichain deployment, new challenges will arise in integrating these aspects with other products. We will actively participate in coordination and consulting to support the streamlined and successful deployment of these features.
+
+Where practical, sunsetting of legacy v2 markets will be supported through continual parameter adjustments to encourage users to migrate to the latest instance of Aave Protocol.
+
+By addressing these challenges, our team aims to support the DAO in achieving sustainable growth and resilience in a rapidly changing environment.
+
+# Specification
+
+The createStream() method of the IAaveEcosystemReserveController will be called to create two streams when the payload gets executed:
+
+**karpatkey**
+Stream: 250k GHO over 180 days
+Address: `0x58e6c7ab55aa9012eacca16d1ed4c15795669e1c`
+
+**TokenLogic**
+Stream: 250k GHO over 180 days
+Address: `0x3e4A9f478C0c13A15137Fc81e9d8269F127b4B40`
```

Comment:
@LucasWongC As per the latest changes, l would update that we transfer the backdated amount and stream the rest amount.

Also instead of using `IAaveEcosystemReserveController ` I would simple put `Collector` as otherwise could be confusing with the collector controller which we are not using in this case.